packages/prettyhtml/test/cli.js

```diff
@@ -0,0 +1,70 @@
+const spawn = require('child_process').spawn
+const test = require('ava')
+const fs = require('fs')
+const path = require('path')
+const dir = path.join(__dirname, './tmp')
+const cliPath = path.join(__dirname, '../cli.js')
+
+try {
+  fs.mkdirSync(dir)
+} catch (err) {
+  if (err.code !== 'EEXIST') throw err
+}
+
+test.cb('single quotes', t => {
+  fs.writeFile(
+    path.join(dir, 'single-quote.html'),
+    '<div id="foo" class=bar/>\n',
+    err => {
+      if (err) throw err
+      const cp = spawn(cliPath, [
+        'test/tmp/single-quote.html',
+        '--single-quote'
+      ])
+      cp.on('close', function(code) {
+        t.is(
+          fs.readFileSync(path.join(dir, 'single-quote.html'), 'utf8'),
+          "<div id='foo' class='bar'/>\n",
+          'should use single instead of double quotes'
+        )
+        t.end()
+      })
+    }
+  )
+})
+
+test.skip.cb('sort attributes', t => {
+  fs.writeFile(
+    path.join(dir, 'sort-attributes.html'),
+    '<div b="bar" a="foo"/>\n',
+    err => {
+      if (err) throw err
```

Comment:
Use a assertion from ava

Reply:
After switching to writeFileSync these assertions are no longer there. I do have some error handling when creating and deleting the tmp folder, should I use an ava assertion there?

Reply:
Yes